### PR TITLE
Domain cred ui

### PR DESCRIPTION
**Pull request by nilsbehlen**

### CODE REVIEW

File: CredentialProvider/core/CCredential.cpp [hunk 1]
```diff
@@ -915,15 +918,15 @@ HRESULT CCredential::Connect(__in IQueryContinueWithStatus *pqcws)
...
 	/////
-
 	HOOK_CHECK_CRITICAL(Hook::Serialization::EndpointInitialization(), Exit);
 	HOOK_CHECK_CRITICAL(Hook::Serialization::DataInitialization(), Exit);
 
 	HOOK_CHECK_CRITICAL(Hook::Serialization::EndpointLoadData(), Exit);
 	HOOK_CHECK_CRITICAL(Hook::Serialization::EndpointLoadDebugData(), Exit);
 
-	if (Data::Provider::Get()->usage_scenario == CPUS_UNLOCK_WORKSTATION || Data::Provider::Get()->usage_scenario == CPUS_LOGON)
+	if (Data::Provider::Get()->usage_scenario == CPUS_UNLOCK_WORKSTATION || Data::Provider::Get()->usage_scenario == CPUS_LOGON || Data::Provider::Get()->usage_scenario == CPUS_CREDUI)
```
Comment (cornelinux) [inline CredentialProvider/core/CCredential.cpp]: If we do "CPUS_CREDUI" here, can we completely remove the lines above 829 - 835. They claim "we currently do not support CREDUI".?

File: CredentialProvider/core/CProvider.cpp [hunk 2]
```diff
@@ -99,7 +99,7 @@ HRESULT CProvider::SetUsageScenario(
 		hr = S_OK;
 		break;
 
-		//case CPUS_CREDUI: // Though, we are prepared
+	//case CPUS_CREDUI: // Though, we are prepared
```
Comment (cornelinux) [inline CredentialProvider/core/CProvider.cpp]: I think we can remove this one? The CREDUI is supported?

Comment (nilsbehlen) [reply CredentialProvider/core/CProvider.cpp]: Yes i think we could remove this finally

**Comment by cornelinux** [comment]
Looks good to me!

**Pull request merged**